src/Shared/Components/ImageCardAccordian/service.ts

```diff
@@ -0,0 +1,76 @@
+import moment from 'moment'
+import { ROUTES, VulnerabilityType, get, sortCallback } from '../../../Common'
+
+interface LastExecutionResponseType {
+    code: number
+    status: string
+    result: {
+        scanExecutionId: number
+        lastExecution: string
+        appId?: number
+        appName?: string
+        envId?: number
+        envName?: string
+        pod?: string
+        replicaSet?: string
+        image?: string
+        objectType: 'app' | 'chart'
+        scanned: boolean
+        scanEnabled: boolean
+        severityCount: {
+            critical: number
+            moderate: number
+            low: number
+        }
+        vulnerabilities: VulnerabilityType[]
+        scanToolId?: number
+    }
+}
+
+function parseLastExecutionResponse(response): LastExecutionResponseType {
+    const vulnerabilities = response.result.vulnerabilities || []
+    const critical = vulnerabilities
+        .filter((v) => v.severity === 'critical')
```

Comment:
create enum for this

Reply:
Fixed in https://github.com/devtron-labs/devtron-fe-common-lib/pull/96, since this has types file in shared and would save conflict